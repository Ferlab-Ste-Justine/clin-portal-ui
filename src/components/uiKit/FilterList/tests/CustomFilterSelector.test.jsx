import React from 'react';
import renderer from 'react-test-renderer';

import CustomFilterSelector from '../index';

describe('CustomFilterSelector', () => {
  test('should be as the snapshot', () => {
    const component = renderer.create(<CustomFilterSelector filterInfo={{ groups: [] }} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
