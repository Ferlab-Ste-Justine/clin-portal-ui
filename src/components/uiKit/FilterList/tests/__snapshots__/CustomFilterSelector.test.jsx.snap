// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomFilterSelector should be as the snapshot 1`] = `
Array [
  <div>
    <button
      className="ant-btn ant-btn-link"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      <span>
        Tout fermer
      </span>
    </button>
  </div>,
  <section
    className="ant-layout"
  />,
]
`;
